sycl/include/sycl/accessor.hpp

```diff
@@ -2405,6 +2405,12 @@ class __SYCL_SPECIAL_CLASS local_accessor_base :
   ConcreteASPtrType MData;
 
 #else
+public:
+  local_accessor_base()
+      : detail::LocalAccessorBaseHost{/*Size*/ sycl::range<3>{0, 0, 0}, /*Dims*/ 0, /*ElemSize*/ sizeof(DataT)}
```

Comment:
This seems like quite a long line, I wonder why clang-format don't suggest to shorten it?